tasks/lzards-backup/tests/test-index.js

```diff
@@ -579,7 +607,7 @@ test.serial('generateAccessUrl switches correctly based on urlType', async (t) =
   t.true(index.generateCloudfrontUrl.calledOnce);
 });
 
-test.serial('backupGranulesToLzards returns the expected payload', async (t) => {
+test.serial('backupGranulesToLzards returns the expected payload with workflow output type input granule', async (t) => {
```

Comment:
```suggestion
test.serial('backupGranulesToLzards returns the expected payload with "workflow output" type input granule', async (t) => {
```
Or something to make it a little clearer that the input granule is of type "workflow output", not "API".

Reply:
https://github.com/nasa/cumulus/pull/3043/commits/ad477d13003920919959a9deef5c8793b2e4cd09#diff-a27a835ebbe71185fd03ceca39186dde0aa282d5b57f339f07c321c488001fa2R610 Perhaps this?

Reply:
Yeah, perfect.